src/libecalc/core/models/compressor/base.py

```diff
@@ -130,6 +130,7 @@ def evaluate_turbine_based_on_compressor_model_result(
             load_adjusted = np.where(
                 np.asarray(compressor_energy_function_result.power) > 0,
                 np.asarray(compressor_energy_function_result.power)
+                * self.data_transfer_object.energy_usage_adjustment_factor
```

Comment:
I think the skip of using factor here was intended, at least it seems like. There was some confusion between 2 different but very similar "adjustment factors" at some point - verify that this is not the case any more. Perhaps check COMPRESSOR_TABULAR or TABULAR?

Reply:
@olelod do you remember if the adjustment factor was intentionally skipped here? And why?

Reply:
The ADJUSTMENT (CONSTANT and FACTOR) for FACILILITY_INPUTS are mapped to energy_usage_adjustment_constant and energy_usage_adjustment_factor in the dto. The POWER_ADJUSTMENT_CONSTANT is mapped to energy_usage_adjustment_constant but energy_usage_adjustment_factor is defaulted to 1. I can not remember if that was intentional - and I can not see the reason why it should be like that.

Reply:
ok, is it possible to include this in a test to have it covered - and see effect before/after? Also, add this explicitly in changelog to make users aware of this bug - and how this will change results to be correct - also for UAT to have a test to test before/after to verify correctness and expected change :)

Reply:
now, I am thinking that perhaps the reason it was not set was because it was hardcoded to 1? And in that case it doesnt need to be informed, unless users expected it to work?